### ISSUE TITLE
LESS Compilation Error without Errors displayed

### ISSUE THREAD

**Issue by enricosoft** [madskristensen/WebEssentials2013#1559]
Hi,
I'm using VS2013 Update 3 with Web essentials 2.3

When I modify or save a LESS file, the preview is always:
"Compilation Error occurred (see error list to navigate to the error location):"
But no errors in "Errors list" area... I've only some warnings and I configured the plugin to separate errors from warning... 

The compilation of LESS files is automatic when save or modify it and I tried also to compile manually from the "WEB ESSENTIALS" menu...

This problems don't allow me to create .css and .min.css automatically (the creation of these files is enabled)

Can you help me?
Thanks in advance!

**Comment by AdiTheSun** [comment]
yes as thorbeck mentioned, it was the "&" special character for me in one of the parent folder name, that was causing this issue. Pretty strange why webessentials wouldn't work if a special character is present in the full absolute path. Anyways thanks to enricosoft and thorbeck. :+1:

**Comment by sashkanz** [comment]
Didn't help me, I don't have any special characters and still having the issue:
/*

Compilation Error occurred (see error list to navigate to the error location):
Error found:


*/

**Comment by kipusoep** [comment]
+1, also have an ampersand (&) in the path.

**Comment by tjperovich** [comment]
My issue was due to having css expressions. Once I commented out the expression's it was fine. 

Currently upgrading some very old classes. I just copy/pasted my css file into an online compiler and deleted large chunks of the file until the compilation worked. Which lead me to find that expressions were the issue. Luckily the author only used it in one class in a 3k line file.

**Comment by develax** [comment]
I've got an empty error too for this less code:
.block-base() {
    border-color: #E2E2E2;
    border-style: solid;
    border-width: 1px 1px 1px 1px;
}

.side-block-base() {
    .block-base;
    margin-top: 10px;
    padding: 10px 10px;
}

/*

Compilation Error occurred (see error list to navigate to the error location):
Error found:


*/

**Comment by tjperovich** [comment]
@develax you're issue is that you have no base class for the mix-ins to compile into. You just have two mix-ins. You can either set `side-block-base` to be a css class, or you can create a class which incorporates the `side-block-base` mix-in.
1)

    .block-base() {
      border-color: #E2E2E2;
      border-style: solid;
      border-width: 1px 1px 1px 1px;
    }

    .side-block-base {
      .block-base;
      margin-top: 10px;
      padding: 10px 10px;
    }

or 2)

    .block-base() {
      border-color: #E2E2E2;
      border-style: solid;
      border-width: 1px 1px 1px 1px;
    }

    .side-block-base() {
      .block-base;
      margin-top: 10px;
      padding: 10px 10px;
    }

    .third-class {
      .side-block-base;
    }

**Comment by develax** [comment]
Thank you! I guess I understand why it worked in another project - because there was a separate file with .third-class. I'll check it carefully tomorrow.

 But why "Error found:" has empty info line is still a question.

**Comment by tjperovich** [comment]
You're welcome!

The empty 'Error found' is due to the LESS compiler not giving a proper error. Web Essentials 2013 is just displaying the messages that the LESS compiler generates. You can make an issue/post on the LESS repository's page: https://github.com/less/less.js/issues

**Comment by develax** [comment]
In another (old) project with absolutely the same css configuration in Base.less there is no error:
http://gyazo.com/c9116940e5247212595050efba28099a
an even more: it produces an empty Base.css file. In a new project with error no css file is produced.
